api/napi/package.json

```
@@ -24,6 +26,7 @@
     "build": "napi build --platform --release --js rust-module.js --dts rust-module.d.ts && npm run compile",
     "build:debug": "napi build --platform --js rust-module.js --dts rust-module.d.ts && npm run compile",
     "install": "npm run build",
+    "docs": "npm run build && typedoc --hideGenerator --readme cover.md index.ts",
```

Note that there's currently warning in the output. I suggest to fix it and then apply this suggestion:

```suggestion
    "docs": "npm run build && typedoc --hideGenerator --treatWarningsAsErrors --readme cover.md index.ts",
```

---

done